environment.py

```diff
@@ -291,7 +291,8 @@ def env() -> Mapping[str, Optional[str]]:
         # for details. This setting may also be used to drive other scaling choices,
         # like the number of shards in Elasticsearch.
         #
-        'AZUL_INDEXER_CONCURRENCY': '64',
+        'AZUL_CONTRIBUTION_CONCURRENCY': '64',
```

Comment:
Documentation needs to be updated to reflect the splitting of the variables and the change in syntax.